src/main/java/com/faforever/api/utils/JsonApiErrorBuilder.java

```diff
@@ -0,0 +1,23 @@
+package com.faforever.api.utils;
+
+import com.google.common.collect.ImmutableMap;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.Map;
+
+@Setter
+public class JsonApiErrorBuilder {
+
+  private String title;
+  private String source;
+  private String pointer = "";
+
+  public Map<String, Serializable> build() {
+    ImmutableMap source = ImmutableMap.of("pointer", pointer);
+    ImmutableMap<String, Serializable> error = ImmutableMap.of(
+        "title", title,
+        "source", source);
+    return ImmutableMap.of("errors", new ImmutableMap[]{error});
+  }
```

Comment:
This is not JSON-API conform, nor is it well readable.

```
return ImmutableMap.of(
  "errors", new ImmutableMap[] {
    ImmutableMap.of(
    "status", status,
    "source", ImmutableMap.of(
      "pointer", pointer
    )
    "title", title,
    "detail", detail
  )});
```

However, the ApiException should be used, which also supports multiple errors. And it should be handled globally.